src/main/java/seedu/address/logic/parser/ListCommandParser.java

```diff
@@ -11,6 +13,11 @@
  */
 public class ListCommandParser implements Parser<ListCommand> {
 
+    public static final String MESSAGE_MULTIPLE_SORT_FIELDS = "Multiple sort fields are not allowed. "
+            + "Please specify only one sort field.";
+    public static final String MESSAGE_MULTIPLE_REVERSE = "Multiple reverse flags are not allowed.";
+    public static final String MESSAGE_INVALID_FORMAT = "Invalid command format! \n%s";
```

Comment:
You can import this constant from `Messages` instead of declaring it here again.